drf_generators/management/commands/generate.py

```diff
@@ -45,15 +45,16 @@ def handle_app_config(self, app_config, **options):
             views = options['views'] if 'views' in options else False
             urls = options['urls'] if 'urls' in options else False
 
-        elif django.VERSION[1] >= 8:
+        elif django.VERSION[1] >= 8 or (django.VERSION[0] == 2
+                                        and django.VERSION[1] >= 0):
             force = options['force']
             format = options['format']
             depth = options['depth']
             serializers = options['serializers']
             views = options['views']
             urls = options['urls']
         else:
-            raise CommandError('You must be using Django 1.7, 1.8 or 1.9')
+            raise CommandError('You must be using Django 1.7, 1.8, 1.9, 2.0+')
```

Comment:
Could you remove 1.7 and 1.8 while we're at it? Also add 1.10. I seem to have overlooked this on the last update.